samples/android/app/build.gradle

```diff
@@ -17,6 +17,8 @@ android {
         applicationId "org.mozilla.samples.gleancore"
         minSdkVersion rootProject.ext.build['minSdkVersion']
         targetSdkVersion rootProject.ext.build['targetSdkVersion']
+        // Keep the version in sync with gleanExpireByVersion which is
+        // set below, near the end of the file.
         versionCode 1
```

Comment:
I hoped you pick something "fun" :(

Reply:
Yeah, my day turned out to be "fun fun fun" :-P